Sources/TripKitUI/helper/RxTripKit/TKGeocoding+Rx.swift

```diff
@@ -156,7 +156,7 @@ public extension Array where Element == TKAutocompleting {
 
 extension ObservableType {
 
-  static func stableRace<Collection: Swift.Collection>(_ collection: Collection, cutOff: RxTimeInterval = .milliseconds(250), fastSpots: Int = 2, comparer: @escaping (Self.Element, Self.Element) -> Bool) -> Observable<[Self.Element]>
+  static func stableRace<Collection: Swift.Collection>(_ collection: Collection, cutOff: RxTimeInterval = .milliseconds(1000), fastSpots: Int = 3, comparer: @escaping (Self.Element, Self.Element) -> Bool) -> Observable<[Self.Element]>
```

Comment:
@nighthawk For my information, what does `fastSpots` do?

Reply:
This is explained in the body:

```
      // Structure:
      // 1. The race winners, gets to take up to fast spots; this is done
      //    by taking from them until the timeout fires.
      // 2. In parallel, we wait for everyone to complete and merge all their
      //    results.
      // 3. The resulting stream is 2 if they complete before the timeout fires
      //    OR first 1 and then 2 but moving the items in 1 to the front.
      //    => This means that if 2 fires before 1, then we only take 2
```

So from all the results that arrive within 1s, we show the best 3 (the "fast spots"), and when the all the remaining results below once they all have arrived.